content/en/docs/concepts/storage/volumes.md

```diff
@@ -33,10 +33,8 @@ drivers, but the functionality is somewhat limited.
 Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod" text="Pod" >}}
 can use any number of volume types simultaneously.
 Ephemeral volume types have a lifetime of a pod, but persistent volumes exist beyond
-the lifetime of a pod. Consequently, a volume outlives any containers
-that run within the pod, and data is preserved across container restarts. When a pod
-ceases to exist, Kubernetes destroys ephemeral volumes; however, Kubernetes does not
-destroy persistent volumes.
+the lifetime of a pod. When a pod ceases to exist, Kubernetes destroys ephemeral volumes; 
+however, Kubernetes does not destroy persistent volumes.
```

Comment:
How about adding this text, the meaning of which would otherwise be lost:
```
For any kind of volume in a given pod, data is preserved across container restarts.
```
?

Is that laboring the point too much?

Reply:
> Is that laboring the point too much?

Not at all, I think!

Thank you for catching and letting me know an important point.

I will push another commit which adds that text in next week!